packages/runtime/src/swagger/swagger.ts

```diff
@@ -128,6 +128,7 @@ export namespace Swagger {
   }
 
   export type Parameter = BodyParameter | FormDataParameter | QueryParameter | PathParameter | HeaderParameter;
+  export type Parameter3 = Parameter & { deprecated?: boolean };
```

Comment:
I'd prefer to keep this in Parameter and set `x-deprecated` if we are dealing with v2, wdty?

Reply:
Sure, I can tweak this